ci/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+RUN ["apt-get", "update"]
+RUN ["apt-get", "install", "-y", "gcc"]
+RUN ["apt-get", "install", "-y", "g++"]
+RUN ["apt-get", "install", "-y", "cmake"]
+
+WORKDIR /build/
+COPY ./ ./project
```

Comment:
COPY and ADD should always be as far down in the Dockerfile as possible because they easily invalidate the following layers.

Reply:
Moved `Copy` to the bottom.